### PR TITLE
Loadout auto-GAGS

### PR DESCRIPTION
**Pull request by Gandalf2k15**
## About The Pull Request

Just automatically sets GAGS in the loadout for items that support it.

:cl:
add: Items in the loadout menu will now automatically be colourable if they support it.
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/loadouts/loadout_items/_loadout_datum.dm [hunk 1]
```diff
@@ -76,6 +74,8 @@ GLOBAL_LIST_EMPTY(all_loadout_datums)
 		return
 
 	var/list/our_loadout = preference_source.loadout_list
+	var/atom/loadout_atom = item_path
+	var/can_be_greyscale = (initial(loadout_atom.greyscale_config) && initial(loadout_atom.greyscale_colors) && (initial(loadout_atom.flags_1) & IS_PLAYER_COLORABLE_1))
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/loadouts/loadout_items/_loadout_datum.dm]: Just to stay consistent with how you did it for the other one.
```suggestion
	var/can_be_greyscale = !!(initial(loadout_atom.greyscale_config) && initial(loadout_atom.greyscale_colors) && (initial(loadout_atom.flags_1) & IS_PLAYER_COLORABLE_1))
```

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
Aside from this minor nitpick, looks good to me!

**Comment by ghost** [comment]
~~Wouldnt this cause some items to be recolorable when they shouldnt be? Such as our extra prisoner jumpsuits (which I'm going to make GAGS like their parent jumpsuit), or color-preset items
I just dont see the point of this system, the other worked fine~~
Nevermind I noticed the is_player_colorable flag, the above is me being dumb

But, wouldnt this cause issues if upstream adds a new gags item to the loadout, with us needing to manually remove the variable?

**Comment by Gandalf2k15** [comment]
> ~Wouldnt this cause some items to be recolorable when they shouldnt be? Such as our extra prisoner jumpsuits (which I'm going to make GAGS like their parent jumpsuit), or color-preset items I just dont see the point of this system, the other worked fine~ Nevermind I noticed the is_player_colorable flag, the above is me being dumb
> 
> But, wouldnt this cause issues if upstream adds a new gags item to the loadout, with us needing to manually remove the variable?

the loadout is not an upstream thing

**Pull request merged**